doc/filters/length.rst

```diff
@@ -9,3 +9,15 @@ the length of a string:
     {% if users|length > 10 %}
         ...
     {% endif %}
+
+.. note::
+
+    For objects implementing the ``__toString()`` magic method, the result is the
+    length of the string being returned.
+
+    If ``Countable`` is implemented as well, the return value of the ``count()`` method
+    takes precedence.
+
+.. versionadded:: 1.33
```

Comment:
@fabpot Do you have tooling/policy/processes in place to make this read 2.3 when merged into the 2.x branch?